pkg/sql/distsql_physical_planner.go

```diff
@@ -3219,11 +3219,19 @@ func (dsp *DistSQLPlanner) createPlanForExport(
 		return PhysicalPlan{}, err
 	}
 
+	// read value of compression codec, default is no compression
+	// otherwise lookup from supported set of codec values
+	compessionCodec := execinfrapb.CSVWriterSpec_none
+	if codec, exist := execinfrapb.CSVWriterSpec_Compression_value[n.codecName]; exist {
```

Comment:
eh, i'm wary of this map lookup. mentioned below, but i'd rather we just use the enum type end-to-end, converting directly to it where to do the initial string validation and avoid this conversion step here.

Reply:
done